package.json

```diff
@@ -33,7 +33,7 @@
       "src/**/*.js"
     ],
     "require": [
-      "babel-register"
+      "@babel/register"
```

Comment:
oh I forgot about this in the nyc config.  Huh.  In any case if this is necessary the `--require @babel/register` on the command line should be redundant